pkg/server/wireexts_oss.go

```diff
@@ -88,6 +88,8 @@ var wireExtsBasicSet = wire.NewSet(
 	pluginsintegration.WireExtensionSet,
 	publicdashboardsService.ProvideServiceWrapper,
 	wire.Bind(new(publicdashboards.ServiceWrapper), new(*publicdashboardsService.PublicDashboardServiceWrapperImpl)),
+	auth.ProvideEmbeddedKeyService,
+	wire.Bind(new(auth.KeyService), new(*auth.EmbeddedKeyService)),
```

Comment:
If we don't have a enterprise version of this we need to put it in wire.go, otherwise this would prevent enterprise from being built